src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -446,7 +446,12 @@ public static NamespaceType Create(string aliasName, ResourceScope resourceScope
         {
             return new NamespaceType(
                 aliasName,
-                Settings,
+                 new NamespaceSettings(
```

Comment:
nit: extra whitespace
```suggestion
                new NamespaceSettings(
```